repositories.bzl

```diff
@@ -1301,9 +1301,9 @@ def com_googleapis_gapic_generator_go_repositories():
             "-exclude=**/testdata",
         ],
         importpath = "google.golang.org/protobuf",
-        sum = "h1:8Ar7bF+apOIoThw1EdZl0p1oWvMqTHmpA2fRTyZO8io=",
+        sum = "h1:6A3ZDJHn/eNqc1i+IdefRzy/9PokBTPvcqMySR7NNIM=",
         # TODO(https://github.com/googleapis/gapic-generator-go/issues/1608): Don't hard-code old version
-        version = "v1.35.2",
+        version = "v1.36.4",
```

Comment:
See changes in 1.36.0 such as Opaque API that will be brought in with this update: https://github.com/protocolbuffers/protobuf-go/releases/tag/v1.36.0